colossalai/fx/tracer/meta_patch/patched_function/torch_ops.py

```diff
@@ -24,6 +24,11 @@ def torch_arange(*args, **kwargs):
     return torch.empty((end - start) // step, dtype=dtype, device="meta")
 
 
+@meta_patched_function.register(torch.finfo)
+def torch_finfo(*args, **kwargs):
+    return torch.finfo(*args)
```

Comment:
kwargs is not passed to torch.finfo?

Reply:
Yes, I will delete the kwargs argument.

Reply:
Should it be `torch.finfo(*args, **kwargs)`?